README.md

```diff
@@ -20,37 +20,20 @@ Please join the discussion on the Angle Community [Discord Server](https://disco
 
 ## Introduction to Angle
 
-### 🏅 Over-Collateralized, Decentralized, and Capital-Efficient Stablecoin Protocol
+Angle is a decentralized stablecoin protocol composed of smart contracts mostly deployed on the Ethereum blockchain. 
 
-Angle is the first over-collateralized, decentralized and capital-efficient stablecoin protocol. It offers full convertibility between stable assets and collateral at oracle value: you can swap 1 of collateral against 1 of stablecoin and conversely.
```

Comment:
Let's try to keep this sentence above, and potentially link it to the Core Module: it's important for people to showcase this: capital-efficiency + over-collateralized

Reply:
Ok yeah, we can specify that it relates to the Core Module then
But in general I think we should change our tagline as it doesn't represent the protocol as a whole anymore